template/$PROJECT_NAME$/.travis.yml

```diff
@@ -0,0 +1,15 @@
+<% MixTemplates.ignore_file_unless(assigns[:ci] == "travis") %>
+language: elixir
+
+elixir:
+  - 1.6.1
+
+addons:
+  postgresql: '9.4'
+
+services:
+  - postgresql
+
+script:
+  - bin/setup
+  - bin/test
```

Comment:
One last change: `bin/setup` already runs `bin/test`, so this line is unnecessary and will make your builds take longer.